src/core/src/core_logic/PatchInstaller.py

```diff
@@ -19,6 +19,7 @@
 import os
 import time
 from core.src.bootstrap.Constants import Constants
+from core.src.service_interfaces.TelemetryWriter import TelemetryWriter
```

Comment:
TelemetryWriter already exists in this context and does not need to be imported.

Please use self.telemetryWriter that is instantiated first on line 32 and is available for the lifetime of this class instance.

Reply:
Fixed.